phpunit/class-wp-rest-url-details-controller-test.php

```diff
@@ -303,7 +262,7 @@ function( $args, $url ) {
 		rest_get_server()->dispatch( $request );
 
 		// Check the args were filtered as expected.
-		$this->assertArraySubset(
```

Comment:
`assertArraySubset` is deprecated in PHPUnit 9+. To ensure the tests work across multiple versions, replacing this assertion.